extern/portable-snippets/debug-trap.h

```diff
@@ -0,0 +1,83 @@
+/* Debugging assertions and traps
```

Comment:
this probably should be placed inside `libs/common` because it's already managed by our cmake system

Reply:
Done.